app/models/manageiq/providers/infra_conversion_job.rb

```diff
@@ -0,0 +1,167 @@
+class ManageIQ::Providers::InfraConversionJob < Job
+  POLL_CONVERSION_INTERVAL = 60
+  POLL_CONVERSION_MAX = 24 * 60 # i.e. default 24 hour
+  POLL_POST_STAGE_MAX = 30 # i.e. default 30 minutes
+
+  def self.create_job(options)
+    # TODO: from settings/user plan settings
+    options[:conversion_polling_interval] = POLL_CONVERSION_INTERVAL
+    options['poll_conversion_max'] = POLL_CONVERSION_MAX
+    options['poll_post_stage_max'] = POLL_POST_STAGE_MAX
```

Comment:
Is there a reason we are mixing strings and symbols?

Reply:
for the `polling_timeout` method
yeah it's ugly, let me try again...

Reply:
done